urbansearch/logging/logutils.py

```diff
@@ -0,0 +1,105 @@
+"""
+Provide utils to enable logging. Example use:
+
+LOGGER = logutils.getLogger(__name__)
+logutils.add_handlers(LOGGER)
+logutils.set_loglevel(LOGGER, logutils.logging.DEBUG)
+
+def function_example:
+    Logger.debug("Debug statement..")
+
+
+"""
+
+import os
+import logging
+import logging.handlers
+
+# TODO Move to config? And absolute/relative choice
+directory = './log/'
+
+
+def _create_directory(dir_path):
+    # Create logging directory if non-existent
+    try:
+        os.makedirs(dir_path, exist_ok=True)
+    except OSError:
+        # TODO do something useful? Report error to user?
+        raise
+
+_create_directory(directory)
```

Comment:
Wouldn't it be better to use something like this? I don't think a function is necessary at all.

    if not os.path.exists(PATH):
        os.makedirs(PATH)